src/js/tabs.js

```diff
@@ -79,130 +75,135 @@
 			tablist.appendChild( li );
 		}
 
-		tabbed.insertBefore( tablist, panels[0] );
+		tabbed.insertBefore( tablist, panels[ 0 ] );
 
 		// Add event listeners to tabs
-		tabs = tablist.querySelectorAll( 'a' );
+		const tabs = tablist.querySelectorAll( 'a' );
 		for ( i = 0; i < tabs.length; i++ ) {
-			tabs[i].addEventListener(
+			tabs[ i ].addEventListener(
 				'click',
 				function( e ) {
 					handleClick( e, tabs, tablist, panels );
 				}
-				);
-			tabs[i].addEventListener(
+			);
+			tabs[ i ].addEventListener(
 				'keydown',
 				function( e ) {
 					handleKeystroke( e, tabs, panels );
 				}
-				);
+			);
 		}
 
 		// Add tab panel semantics and hide them all
 		for ( i = 0; i < panels.length; i++ ) {
-
-			panels[i].setAttribute( 'role', 'tabpanel' );
-			panels[i].setAttribute( 'tabindex', '-1' );
-			panels[i].setAttribute( 'aria-labelledby', tabs[i].id );
-			panels[i].hidden = true;
-
+			panels[ i ].setAttribute( 'role', 'tabpanel' );
+			panels[ i ].setAttribute( 'tabindex', '-1' );
+			panels[ i ].setAttribute( 'aria-labelledby', tabs[ i ].id );
+			panels[ i ].hidden = true;
 		}
 
 		// Initially activate the first tab and reveal the first tab panel
-		tabs[0].removeAttribute( 'tabindex' );
-		tabs[0].setAttribute( 'aria-selected', 'true' );
-		panels[0].hidden = false;
+		tabs[ 0 ].removeAttribute( 'tabindex' );
+		tabs[ 0 ].setAttribute( 'aria-selected', 'true' );
+		panels[ 0 ].hidden = false;
 
 		// Go to the anchored tab, if there is one
-		anchor = getAnchoredElement( tabbed );
+		const anchor = getAnchoredElement( tabbed );
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['getAnchoredElement' was used before it was defined.](https://app.codacy.com/manual/uriweb/uri-component-library/pullRequest?prid=4811013)